mirrors_countme/readers.py

```diff
@@ -64,9 +64,6 @@ def __iter__(self):
         for item in self._iter_rows():
             yield self._itemfactory(item)
 
-    def __contains__(self, item):
```

Comment:
I didn't see this called from anywhere. Might be a candidate for removal?

Reply:
Looks like it!